src/settings.ts

```diff
@@ -5,8 +5,11 @@ export type ActivateCallback = (
   button: HTMLElement
 ) => void
 
+export type DismissCallback = (popover: HTMLElement) => void
```

Comment:
As a sidenote, if we have a `dismissCallback` with the same signature as `activateCallback`, we would be able to use a single `ActionCallback` for both.